### DISCUSSION TITLE
Lightning Network Donations

### DISCUSSION THREAD

**Discussion by hyperfree** [elementary/triage#408] [Ideas]
Many users of Elementary OS have never donated a cent to the developers of the OS or those of individual apps. One reason is that credit cards are cumbersome to use. Further, card issuers don't respect user privacy very much. Lastly, some people may don't have credit cards because they're excluded from the financial system.

A great solution would be to provide an optional way to send value via the Lightning Network.

The Lightning Network is an open protocol to transfer value over the internet. It's inclusive since it doesn't require you to have a bank account. This means anyone can send and receive value, regardless of nationality and geographic position. More and more app developers have started to accept donations via the Lightning Network because it's easier to integrate compared to card processing solutions. Also many [game developers](https://dca-signals.com/earn-bitcoin-playing-games/) have started using Lightning as a way to reward users. 

Lightning would be an excellent alternative to credit cards. Credit cards are exclusive to people who are privileged to be granted banking access. Further credit card processing often comes with big fees. On the Lightning Network, value can be transferred with minimal fees.

[Here's a good article about the "value enabled web"](https://dergigi.com/2022/12/18/a-vision-for-a-value-enabled-web/)

Another example of how Lightning is successfully used to enable donations is the crowdfunding platform [geyser.fund](https://geyser.fund)

Allowing donations over the lightning network can open a new stream of revenue for developers, provide a more privacy aware alternative to credit cards and set a positive example for financial inclusion.

**Comment by danirabbit** [comment]
Hey thanks for your idea. There's no current plan to accept crypto currency, however we support all of the payment methods we can through Stripe which includes not just credit cards but Apple Pay and Google Pay, Link, Bank Transfer, CashApp Pay, AliPay, and more.

As a US company we must comply with US law including embargo etc, so it would not be legal for us to have a goal of taking payment regardless of nationality or location. We have to work with payment methods that allow us to comply with the law.
